src/main/java/org/broadinstitute/hellbender/tools/walkers/genotyper/afcalc/AlleleFrequencyCalculator.java

```diff
@@ -77,7 +77,9 @@ public static AlleleFrequencyCalculator makeCalculator(final DragstrParams drags
         final double[] log10Likelihoods;
         if (g.hasLikelihoods()) {
             log10Likelihoods = g.getLikelihoods().getAsVector();
-        } else if (g.isHomRef()) {
+        } else if ( g.isHomRef()) {
+            //no-call with no PLs seems risky, but there are a few places in the QUAL/AF code where we subset alleles,
+            // but then leave the genotypes as no-calls
```

Comment:
This comment still seems confusing/misplaced. It talks about handling no-calls in the case that now handles hom-ref (no-calls now fall through to the `else` block below).

Reply:
Fixed.